serverr/satabase service/databaseservice.js

```diff
@@ -0,0 +1,47 @@
+var { usersModelColletion } = require('../userschema');
+
+
+var findUserByEmail = async (email) => {
+    var check = await usersModelColletion.findOne({ email: email }, (error, result) => {
+        if (!error) {
+            return result
+        }
+    })
+    return check;
+}
+
+
+var saveUser = (newUser, response) => {
+    newUser.save((error, result) => {
+        !error ? response.send({ message: 'User register' }) :
+            response.send({ message: 'Some problem' })
+    })
+}
+
+
+
+var upDateUserPassword = (email, hashPassword, response) => {
+    usersModelColletion.updateOne({ email: email }, { password: hashPassword }, (error, success) => {
+        success ? response.send({ message: 'Your password is update' }) :
+            response.send({ message: 'Something wrong' })
+    })
+}
+
+
+
+var deleteUserAccount = (email, respone) => {
+    usersModelColletion.remove({ email: email }, (error, success) => {
+        if (success) respone.send({ message: 'User deleted' })
+    }).catch((error) => respone.send({ message: 'Somthing wrong' }))
```

Comment:
```suggestion
    }).catch((error) => respone.status(404).send({ message: 'Somthing wrong' }))
```
what about sending status codes with your messages?
See this list: https://httpstatuses.com/
add status code to all errors.